### ISSUE TITLE
About the loss in CVAE-GAN

### ISSUE THREAD

**Issue by zhanglonghao1992** [tatsy/keras-generative#15]
Hi, your implementation is really fascinating~
I have some questions about the loss function part of CVAE-GAN. In the paper, L_GD and L_GC have the loss weight 1e-3, but it is not found in your implementation. Does this affect the training of the generator? I have trained 30 epochs on celebB dataset, the generator accuracy is always 0.
Looking forward your reply.

**Comment by tatsy** [comment]
Hi @zhanglonghao1992,

I don't remember how I set the training parameters in this implementation but I guess I adjusted them by testing several simple datasets such as MNIST and SVHN.

So, maybe you need to tune again by yourself to other dataset using my code. Unfortunately, these codes particularly that for CVAE-GAN is not tested carefully with large datasets such as CelebA, you might need to further modify some parts of codes for making it work properly for them.

Thanks for your attention to my code!
